technical-reports/resolver/terminology.md

```diff
@@ -1,103 +1,37 @@
-# Resolver Terminology
+# Terminology
 
-<dl>
-<dt>Resolver</dt>
-<dd>
+## <dfn>Resolver</dfn>
 
-A mechanism that combines token sets and applies modifiers to produce a final, resolved set of design tokens.
+The mechanism by which multiple possible values of design tokens are reduced to a single value, i.e. this module.
 
-</dd>
-<dt>Token Set</dt>
-<dd>
+## <dfn>Set</dfn>
 
-A collection of design tokens grouped together, often representing foundational tokens, component-specific tokens, or theme-specific tokens.
+Alternate design token values that are defined in [&lt;token-defs&gt;](#token-defs), intended to be merged in a specific order.
 
-</dd>
-<dt>Dimension</dt>
-<dd>
+## <dfn>Modifier</dfn>
 
-Dimensions are categories used to organize your design tokens. See them as contexts in which token values might change:
+Alternate design token value accessed with an [=input=], that takes priority over a [=set=]. Modifiers MAY be [=orthogonal=].
```

Comment:
Could probably use some help here 😅. I wanted to build off the idea of sets being **alternate versions of a design token** and also communicate these all have unique identifiers in a key–value relationship. They also take priority.

None of these may be _definitions,_ though, so thoughts welcome 🙏